.github/ISSUE_TEMPLATE.md

```diff
@@ -1,51 +1 @@
-<!--- Verify first that your issue is not already reported on GitHub -->
-<!--- Also test if the latest release and devel branch are affected too -->
-<!--- Complete *all* sections as described, this form is processed automatically -->
-
-##### SUMMARY
-<!--- Explain the problem briefly below -->
-
-##### ISSUE TYPE
-<!--- Pick one below and delete the rest -->
- - Bug Report
- - Feature Idea
- - Documentation Report
-
-##### COMPONENT NAME
-<!--- Write the short name of the module, plugin, task or feature below, use your best guess if unsure -->
-
-##### ANSIBLE VERSION
-<!--- Paste verbatim output from "ansible --version" between quotes -->
-```paste below
-
-```
-
-##### CONFIGURATION
-<!--- Paste verbatim output from "ansible-config dump --only-changed" between quotes -->
-```paste below
-
-```
-
-##### OS / ENVIRONMENT
-<!--- Provide all relevant information below, e.g. target OS versions, network device firmware, etc. -->
-
-##### STEPS TO REPRODUCE
-<!--- Describe exactly how to reproduce the problem, using a minimal test-case -->
-
-<!--- Paste example playbooks or commands between quotes below -->
-```yaml
-
-```
-
-<!--- HINT: You can paste gist.github.com links for larger files -->
-
-##### EXPECTED RESULTS
-<!--- Describe what you expected to happen when running the steps above -->
-
-##### ACTUAL RESULTS
-<!--- Describe what actually happened. If possible run with extra verbosity (-vvvv) -->
-
-<!--- Paste verbatim command output between quotes -->
-```paste below
-
-```
+Please raise issues via the [new interface](https://github.com/ansible/ansible/issues/new/choose)
```

Comment:
After this, all the comments made by ansibot now point to the wrong template.

cc @gundalow ^